testing/kfctl/kfctl_go_test.py

```diff
@@ -91,6 +92,15 @@ def test_build_kfctl_go(app_path, project, use_basic_auth, use_istio):
     if os.getenv("PULL_PULL_SHA"):
       pull_manifests = "@" + os.getenv("PULL_PULL_SHA")
 
+  # We need to specify a valid email because
+  #  1. We need to create appropriate RBAC rules to allow the current user
+  #     to create the required K8s resources.
+  #  2. Setting the IAM policy will fail if the email is invalid.
+  email = util.run(["gcloud", "config", "get-value", "account"])
```

Comment:
Does this work with service accounts?

Reply:
I think so. This is original code, I just move it.